scripts/masterCron.ts

```diff
@@ -211,6 +249,47 @@ async function getAnnualizedVol(underlying: string, resolution: number) {
   return candles[candles.length - 1][pricePoint] * 10 ** 6;
 }
 
+async function updateTokenList(
+  fileName: string,
+  vaultArtifactAbi: any,
+  ierc20Abi: any,
+  provider: any,
+  network: string
+) {
+  let tokens = [];
+
+  let token = {
+    chainId: 1,
+    address: "",
+    name: "",
+    symbol: "",
+    decimals: 0,
+  };
+
+  for (let vaultName in deployments[network].vaults) {
+    const vault = new ethers.Contract(
+      deployments[network].vaults[vaultName].address,
+      vaultArtifactAbi,
+      provider
+    );
+
+    let oToken = new ethers.Contract(
```

Comment:
Since we're appending to the list of tokens in the file, we need to verify that the otoken doesn't already exist in the list

https://github.com/ribbon-finance/ribbon-v2/pull/119/files#r706706095

Reply:
see above